test/ng/httpSpec.js

```diff
@@ -2345,6 +2345,10 @@ describe('$http param serializers', function() {
       expect(jqrSer({someDate: new Date('2014-07-15T17:30:00.000Z')})).toEqual('someDate=2014-07-15T17:30:00.000Z');
     });
 
+    it('should NOT serialize functions', function() {
+      expect(defSer({foo: 'foov', bar: function() {}})).toEqual('foo=foov');
+    });
```

Comment:
This is not common functionality. It does not belong here, but in `default array serialization` section.